be/btree.c

```diff
@@ -1256,12 +1256,14 @@ static void btree_truncate(struct m0_be_btree *btree, struct m0_be_tx *tx,
 			 * node->bt_child_arr[node->bt_num_active_key]
 			 * leaf node
 			 */
-			if (i > 0)
+			if (i > 0) {
 				btree_pair_release(btree, tx,
 						&parent->bt_kv_arr[i-1]);
 
-			if (limit > 0)
-				limit--;
+				if (limit > 0)
```

Comment:
On LINE 1216 and 1226, the `limit` is checked before calling  `btree_pair_release()`.

But here, the `btree_pair_release()` is called regardless the value of `limit`.

Is it possible to overflow the TX limit?